src/wasm/kotlin/java/arcs/BUILD

```diff
@@ -8,12 +8,13 @@ KOTLINC_OPTS = [
     "-Xuse-experimental=kotlin.ExperimentalMultiplatform",
 ]
 
-kt_jvm_library(
+
+arcs_kt_jvm_library(
     name = "arcs",
-    srcs = glob(
-        ["**/*.kt"],
-        exclude = ["wasm/**"],
-    ),
+    srcs = glob([
+        "*.kt",
+        "**/jvm/*.kt",
+    ]),
```

Comment:
Can you update the wasm rule below to match this logic? (include wasm/**, rather than exclude jvm/**)